src/pybind/mgr/cephadm/module.py

```diff
@@ -1415,17 +1421,146 @@ def _create_osd(self, host, cmd):
 
         return "Created osd(s) on host '{}'".format(host)
 
+    def create_osds(self, drive_group):
+        """
+        Create a new osd.
+
+        The orchestrator CLI currently handles a narrow form of drive
+        specification defined by a single block device using bluestore.
+
+        :param drive_group: osd specification
+
+        TODO:
+          - support full drive_group specification
+          - support batch creation
+        """
+
+        return self.get_hosts().then(lambda hosts: self._create_osd(hosts, drive_group))
+
+    def _remove_osds_bg(self):
```

Comment:
Would it make sense to have a dedicated python module for this one? Everything ends up in one big file here.

Reply:
I have nothing against it.. any preferences for the name?

Reply:
well, maybe we should move it in a followup PR as we'd also move the upgrade bits (possibly more?), mh?